Src/MoneyFox.DataAccess/DataServices/PaymentService.cs

```diff
@@ -129,6 +129,7 @@ public async Task<IEnumerable<Payment>> GetPaymentsByAccountId(int accountId)
                 {
                     var list = await dbContext.Payments
                         .Include(x => x.Category)
+						.Include(x=>x.RecurringPayment)
```

Comment:
is there a reason for this indent?

Reply:
probably forgot to convert spaces to tabs or something, will be fixing

Reply:
resolved in latest updates